### DISCUSSION TITLE
Rare Hairstyle

### DISCUSSION THREAD

**Discussion by M708591** [Prodigy-Hacking/ProdigyMathGameHacking#943] [(A) Anything]
I tried to do Control Shift i on my Prodigy account but nothing happened! Can someone please help me?! It's so confusing and I might need someone to hack into my account and do it for me. Pls, I  REALLY REALLY REALLY NEED IT! I also need some help getting a rare hairstyle.

**Comment by LeoBadeaux** [comment]
I would lock it but I'll let it slide and let a community member help you

**Comment by M708591** [comment]
If anyone can help me, please help me immediately when you have the chance.

**Comment by NX721** [comment]
> I tried to do Control Shift i on my Prodigy account but nothing happened! Can someone please help me?! It's so confusing and I might need someone to hack into my account and do it for me. Pls, I REALLY REALLY REALLY NEED IT! I also need some help getting a rare hairstyle.

If you can't do Ctrl Shift I, it's either disabled from an admin.
